plugins/module_utils/ndb/time_machines.py

```diff
@@ -261,3 +264,59 @@ def _build_spec_schedule(self, payload, schedule):
 
         payload["schedule"] = schedule_spec
         return payload, None
+
+    def get_default_data_access_spec(self, override_spec=None):
+        spec = deepcopy({"nxClusterId": "", "type": "OTHER", "slaId": ""})
+        if override_spec:
+            for key in spec.keys():
+                if override_spec.get(key):
+                    spec[key] = deepcopy(override_spec[key])
+
+        return spec
+
+    def get_data_access_spec(self, old_spec=None):
```

Comment:
assign build_spec_methods here with builders for DAM before as per required builders of data access management spec